R/execute.R

```diff
@@ -110,10 +110,12 @@ execute_array_udf <- function(namespace, array, udf, layout=NULL, selectedRanges
   # object.
   if (typeof(resultObject) != "raw") {
     className <- class(resultObject)[1]
+    print(resultObject)
```

Comment:
Printing the entire object _may_ not be a good idea...  But it is hard to say.

Reply:
@eddelbuettel agreed

I like this better:

```
  bodyAsJSON <- rawToChar(resp$content)
  bodyParsed <- jsonlite::fromJSON(bodyAsJSON)
  ApiResponse$new(paste("Server returned" , httr::status_code(resp) , "response status code. Message:\n", 
    bodyParsed$message), resp)
```

giving

```
tiledbcloud: received error response:
Server returned 500 response status code. Message:
 json: cannot unmarshal string into Go value of type []json.RawMessage
```